numba_dppy/config.py

```diff
@@ -13,20 +13,36 @@
 # limitations under the License.
 
 import os
+import warnings
+from packaging import version
 
-
+# Check for dpctl 0.7.0 or higher on the system.
+_dpctl_found = False
 try:
     import dpctl
 
-    dppy_present = False
-    try:
-        # For the extension to work we should have at least one
-        # non-host SYCL device.
-        dppy_present = not dpctl.select_default_device().is_host
-    except ValueError:
-        dppy_present = False
+    # Versions of dpctl lower than 0.7.0 are not compatible with current main
+    # of numba_dppy.
+    if version.parse(dpctl.__version__) < version.parse("0.7.0"):
+        msg = "numba_dppy requires dpctl 0.7.0 or higher."
+        warnings.warn(msg, UserWarning)
```

Comment:
Is generating an warning the intended behavior? Will users be allowed to proceed when version is lower than 0.7.0?